zio-http/src/main/scala/zio/http/codec/QueryCodecs.scala

```diff
@@ -29,4 +29,28 @@ private[codec] trait QueryCodecs {
   def queryAs[A](name: String)(implicit codec: TextCodec[A]): QueryCodec[A] =
     HttpCodec.Query(name, codec)
 
+  /**
+   * Alias to query
+   */
+  def paramStr(name: String): QueryCodec[String] =
```

Comment:
The motivation behind this naming was, it is confusing to encode endpoint as 

```scala

Endpoint.get(path).query(query("name"))

```

vs

```scala
Endpoint.get(path).query(paramStr("name"))

```

Reply:
In fact with one of the suggestions above, it will be:

```scala
Endpoint.get(path :? paramStr("name"))

```

Reply:
If these are going to change, then `queryAs` should change as well, to `paramAs`.

Reply:
Or maybe just delete `queryAs` since you have `param` below.

Reply:
I didn't want to break compatibility by deleting `queryAs`, unless we are ok with it.

I can change `param` to `paramAs`.